P5/Source/Guidelines/en/CMC-ComputerMediatedCommunication.xml

```diff
@@ -0,0 +1,1326 @@
+          screen) and transmitted over a computer network (typically the
+          internet).</item>
+      </list>
+      Such communications may be expressed as posts (cf. <ptr
+        target="#CMCcmcpost"/>), utterances, onscreen activities, or bodily activities exerted
+      by a virtual avatar. 
+    </p>
+    <p>The following kinds of platforms support CMC: <list rend="simple">
+        <item>chats, messengers, or online forums;</item>
+        <item>social media platforms and applications;</item>
+        <item>the communication functions of collaborative platforms and projects (e.g. an online
+          learning environment, or a <soCalled>talk</soCalled> page);</item>
+        <item>3D virtual world environments;</item>
+        <item>other interactive services supported by the internet.</item>
+      </list></p>
+    <p>CMC supports multimodal expression combining text, images, sound. Whereas early CMC systems
+      (e.g. Internet Relay Chat, <soCalled>IRC</soCalled> for short, the Usenet
+      <soCalled>newsgroups</soCalled>, or even the Unix <name>talk</name> system)
+      were completely ASCII-based, most CMC applications now permit combining media formats (e.g. written or spoken
+      language with graphic icons and images) and mixing communication technologies on one platform (e.g. combined use of an audio connection, a chat system, and a
+      3D interface in which users control a virtual avatar as in many multiplayer online computer
+      games or in virtual worlds).</p>
+  </div>
+  <div type="div2" xml:id="CMCUnits">
+    <head>Basic Units of CMC</head>
+    <p>This section describes the encoding mechanisms for the basic units of CMC and for their
+      combined use to encode CMC data.</p>
+    <p>We use the term <term>basic CMC unit</term> to refer to a communication produced by an interlocutor to initiate or contribute to an ongoing
+      CMC interaction or joint CMC activity. Contributions
+      to an ongoing interaction are produced to perform a move to develop
+      the interactional sequence, for instance to respond in chats or forum discussions. Contributions to joint
+      CMC activities may not all be directly interactional; some may be part of a collaborative
+      project of the involved individuals. Such collaboration could involve editing activities in a shared text editor
+      or whiteboard in parallel with an ongoing CMC interaction (chat, audio conversation, or
+      audiovideo conference) in the same CMC environment in which these editing activities are
+      discussed by the participants.</p>
+    <p>Basic units of CMC can be described according to three criteria: 
+      <list rend="numbered">
+        <item>the temporal properties
+          of when these contributions are produced by their creators, transmitted via CMC systems, and
+          made accessible for the recipients;</item>
+        <item>the modality of the unit as a whole, whether verbal or noverbal;</item>
```

Comment:
noverbal (in the graphic above, it is non-verbal, further down it is nonverbal). this should be normalized.